docs/vscode_install.md

```diff
@@ -1,8 +1,8 @@
 # Installation in VS Code
 
-Vapoursynth Preview can be used in a number of different ways.<br>One of them is directly in Microsoft's free code editor [Visual Studio Code](https://code.visualstudio.com).
+vs-preview can be used in a number of different ways.<br>One of them is directly in Microsoft's free code editor [VS Code](https://code.visualstudio.com).
```

Comment:
why did you feel the need to shorten visual studio as vs? that can be confused with vapoursynth